
<launch> 
 <node pkg="planning_environment" type="clear_known_objects" name="laser_clear_objects" output="screen">
    <remap from="cloud_in" to="tilt_scan_cloud" />
    <remap from="cloud_out" to="tilt_scan_cloud_known" />
    <param name="sensor_frame" type="string" value="laser_tilt_mount_link" />			
    <param name="fixed_frame" type="string" value="base_link" />
    <param name="object_padd" type="double" value=".02" />
    <param name="object_scale" type="double" value="1.0" />
  </node>
</launch>
